tests/features/frontend_cache_purge.feature

```diff
@@ -515,9 +515,9 @@ Feature:
 
   Scenario: No purge are instructed if a part of the configuration has disappeared.
     Given the following cache purge rules:
-      | Content Type | Paths to Purge         |
-      | page         | /more-basic-pages, /   |
-      | page         | /even-more-basic-pages |
+      | Content Type | Paths to Purge       |
+      | page         | /more-basic-pages, / |
+      | page         |                      |
```

Comment:
Hello,

I changed this part because of the remark of @voidtek. I realized that I needed 2 rules to highlight the fact whatever the number of rules, nothing is sent to Varnish if the settings are not complete.

IMO, the previous version was not pertinent and I think in an other ticket we should align the scenario above this one, because it implies 2 rules that impact other pages than the currently saved page.

- /even-more-basic-pages
- / (front page)
- /more-basic-pages

So, the test covered one rule type only.

By replacing the rule with the "/even-more-basic-pages" path by the new one that purge , I cover now the 2 possible rule types:

1.  Purge of a specific page when a page content is saved; it is the first rule described in the table;
2. Purge the currently saved page; it is the second rule of the table.

The test is then complete for me.